suite-native/atoms/src/Pictogram/CriticalShapeSvg.tsx

```diff
@@ -0,0 +1,22 @@
+import Svg, { Path } from 'react-native-svg';
+
+import { useNativeStyles } from '@trezor/styles';
+
+export const CriticalShapeSvg = () => {
+    const { utils } = useNativeStyles();
+    const borderColor = utils.colors.backgroundAlertRedSubtleOnElevationNegative;
+    const backgroundColor = utils.colors.backgroundAlertRedSubtleOnElevation2;
+
+    return (
+        <Svg width={112} height={112} fill="none">
```

Comment:
same here

Reply:
Similar comment as above. I thought about it but I don't se an actual benefit in introducing a constant. I'd just create false expectations.